### PR TITLE
Add links to OBS projects to install documentation

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
This fixes a broken link and adds links to the certain projects in OBS
to have no need for a manual search.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
The compatibility matrix below my code changes needs to be updated to mention 1.19 and 1.20. I saw that builds for Centos 7 for example are existing, so I was not sure if we want to mention them as "to be supported". Any thoughts on that?
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by fidencio** [comment]
@saschagrunert. thanks for the PR, it does look good to me.

While here, already touching this part, wouldn't  make sense to also update the provided version of CRI-O in the table right below your changes?  It shows 1.18 as the latest avaiable version there. (This is a suggestion only, not a **must** ;-)).

**Comment by saschagrunert** [comment]
> @saschagrunert. thanks for the PR, it does look good to me.
> 
> While here, already touching this part, wouldn't make sense to also update the provided version of CRI-O in the table right below your changes? It shows 1.18 as the latest avaiable version there. (This is a suggestion only, not a **must** ;-)).

Yes, I had a concern while trying to update the table:

> The compatibility matrix below my code changes needs to be updated to mention 1.19 and 1.20. I saw that builds for Centos 7 for example are existing, so I was not sure if we want to mention them as "to be supported". Any thoughts on that?

**Comment by fidencio** [comment]
> Yes, I had a concern while trying to update the table:

One can't properly read in the morning, sorry. :-)

**Comment by saschagrunert** [comment]
No worries @fidencio! :)

/retest

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by haircommander** [comment]
I actually don't think we need the compatibility matrix anymore. we used to include it because we didn't have a work around for go-1.14+ needed in debian, but now we do

**Comment by haircommander** [comment]
/retest

changes LGTM, I should cleanup the subprojects today so users don't get confused (there's a top-level cri-o project that we don't use, and it's listed near the top). but I'll be able to do that before CI gets its act together and merges this :)

**Comment by saschagrunert** [comment]
Rebased on top of the latest master.

File: install.md [hunk 1]
```diff
@@ -36,10 +36,19 @@ It is assumed you are running a Linux machine.
...
 
+All available versions are available in the OBS [libcontainers:stable
+project](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable),
+which contains
+[subprojects](https://build.opensuse.org/project/subprojects/devel:kubic:libcontainers:stable)
+to the dedicated versions.
+
+Below is a compatibility matrix between versions of CRI-O (y-axis) and
+distributions (x-axis):
+
```
Comment (haircommander) [inline install.md:51]: nit: we dropped the matrix :)

Comment (saschagrunert) [reply install.md]: Opps, removed that comment :+1:

**Pull request closed without merging**